tests/js/spec/components/commitRow.spec.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import {fireEvent, render, screen} from '@testing-library/react';
+
+import {mountWithTheme} from 'sentry-test/reactTestingLibrary';
+
+import {openInviteMembersModal} from 'sentry/actionCreators/modal';
+import {CommitRow} from 'sentry/components/commitRow';
+import {Commit, Repository, User} from 'sentry/types';
+
+jest.mock('sentry/components/hovercard', () => {
+  return {
+    __esModule: true,
+    default: ({body}) => {
+      return body;
+    },
+  };
+});
+
+jest.mock('sentry/actionCreators/modal', () => {
+  return {
+    ...jest.requireActual('sentry/actionCreators/modal'),
+    openInviteMembersModal: jest.fn(),
+  };
+});
+
+const baseCommit: Commit = {
+  dateCreated: '2020-11-30T18:46:31Z',
+  id: 'f7f395d14b2fe29a4e253bf1d3094d61e6ad4434',
+  message: 'ref(commitRow): refactor to fc\n',
+  author: {
+    id: '0',
+    username: 'author',
+    ip_address: '192.168.1.1',
+    email: 'author@commit.com',
+    name: 'Author',
+  } as User,
+  repository: {
+    id: '1',
+    integrationId: '2',
+    name: 'getsentry/sentry',
+    dateCreated: '2019-11-30T18:46:31Z',
+  } as Repository,
+  releases: [],
+};
+
+const getByTextContent = text => {
```

Comment:
maybe you are looking for some helper function like https://github.com/getsentry/sentry/blob/849626fdd4a8ed40111745dfc11e43950db5f312/tests/js/sentry-test/utils.tsx#L16? 🤷‍♀️

Reply:
hm maybe the helper function will need to be updated.

Maybe something like this would solve the issue

```
function findTextWithMarkup(contentNode: null | Element, textMatch: string | RegExp) {
  const hasText = (node: Element) =>
    !!(typeof textMatch === 'string'
      ? node.textContent?.includes(textMatch)
      : node.textContent?.match(textMatch));

  const nodeHasText = hasText(contentNode as Element);
  const childrenDontHaveText = Array.from(contentNode?.children || []).every(
    child => !hasText(child)
  );
  return nodeHasText && childrenDontHaveText;
}

```

Reply:
👍🏼 Didnt know where it was hehe, thanks for pointing it out, I just updated it slightly